contracts/utils/contracts/src/interfaces/IOwnable.sol

```diff
@@ -1,8 +1,32 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
 pragma solidity ^0.5.9;
 
 
 contract IOwnable {
 
+    /// @dev Emitted by Ownable when ownership is transferred.
+    /// @param newOwner The new owner of the contract.
+    event OwnershipTransferred(address newOwner);
```

Comment:
Looking at Zeppelin's implementation of `Ownable` and they use a slightly different event. Might be worth using the same one to standardize things as much as possible (though I don't feel the need to make their other changes): https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/ownership/Ownable.sol

```solidity
event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
```